Mediapipe.Net/Calculators/BlazePoseCpuCalculator.cs

```diff
@@ -9,7 +9,18 @@ namespace Mediapipe.Net.Calculators
 {
     public class BlazePoseCpuCalculator : CpuCalculator<NormalizedLandmarkListPacket, NormalizedLandmarkList>
     {
-        protected override string GraphPath => "mediapipe/graphs/pose_tracking/pose_tracking_cpu.pbtxt";
+        protected override string GraphPath { get; } = "mediapipe/graphs/pose_tracking/pose_tracking_cpu.pbtxt";
+
         protected override string SecondaryOutputStream => "pose_landmarks";
+
+        public BlazePoseCpuCalculator()
+        {
+
+        }
+
+        public BlazePoseCpuCalculator(string graphPath)
+        {
+            GraphPath = graphPath;
+        }
```

Comment:
I think all of this is unnecessary. You should make it overridable in the base `CpuCalculator` and `GpuCalculator`.

Reply:
it's abstract, you can't assign a value to it from the base class

Reply:
You can by making a base constructor

Reply:
huh didn't know that was a thing